owners/src/github.js

```diff
@@ -36,7 +36,7 @@ class PullRequest {
   /**
    * Whether or not the pull request is open.
    *
-   * @return {boolean} true if the pull request is open.
+   * @return {!boolean} true if the pull request is open.
```

Comment:
I think we're not supposed to use `!` for scalar types, or for return types at all? /cc @rsimha

Reply:
See I thought that too re: return types but then on another PR somewhere I got called out for never using it and I couldn't find any docs saying not to so I thought I hallucinated it :thinking:

Reply:
As for scalars, you're right, will remove.

Reply:
Here's a closure compiler example with `@return {!Element}`
https://github.com/google/closure-compiler/wiki/Types-in-the-Closure-Type-System#typedefs

Reply:
Just scalar types. See [this](https://github.com/ampproject/amphtml/blob/master/build-system/eslint-rules/closure-type-primitives.js#L147-L167) custom `amphtml` rule.